api/auth/index.ts

```diff
@@ -56,6 +58,15 @@ export const getXuiNodeMiddleware = () => {
     const idamApiPath = getConfigValue(SERVICES_IDAM_API_URL)
     const s2sSecret = getConfigValue(S2S_SECRET)
     const tokenUrl = `${getConfigValue(SERVICES_IDAM_API_URL)}/oauth2/token`
+    const userName = getConfigValue(SYSTEM_USER_NAME)
+    const password = getConfigValue(SYSTEM_USER_PASSWORD)
+
+    const routeCredential = {
```

Comment:
Can we make this property name a bit more descriptive if it's specific to the caseworker route? At the moment, it's quite generic and I _assume_ it's not going to be used for any other routes...?

Reply:
we want it to be generic. This is a system user name which we are going to use it for several roles/services.

Reply:
But this only seems to apply to a single route. Will we be adding more routes in the future with identical credentials?

Reply:
Yes you got the point. This is open to other routes.